packages/e2e-test-utils-playwright/src/login-user.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Internal dependencies
+ */
+import { WP_USERNAME, WP_PASSWORD } from './shared/config';
+
+/**
+ * Performs log in with specified username and password.
+ *
+ * @this {import('./').TestUtils}
+ * @param {?string} username String to be used as user credential.
+ * @param {?string} password String to be used as user credential.
+ */
+export async function loginUser(
+	username = WP_USERNAME,
+	password = WP_PASSWORD
+) {
+	if ( ! this.isCurrentURL( 'wp-login.php' ) ) {
+		await this.page.goto( this.createURL( 'wp-login.php' ) );
+	}
+
+	await this.page.press( '#user_login', 'Control+A' );
```

Comment:
Instead of using `pressKeyWithModifier` we can just use the key combination syntax provided by playwright. See more info [here](https://playwright.dev/docs/api/class-page#page-press).